docs/developer-docs/web-apps/verifiable-credentials/relying-party.mdx

```diff
@@ -0,0 +1,272 @@
+---
+keywords: [intermediate, tutorial, identity, credentials, verifiable credentials, relying party]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Become a relying party
+
+<MarkdownChipRow labels={["Intermediate", "Tutorial"]} />
+
+## Overview
+
+A **relying party** is a service or app that requests a user's verifiable credentials from the [issuer](issuer.mdx) API. Relying parties use the identity provider API to communicate with the issuer API using an **identity provider** such as Internet Identity. Identity providers must support the [verifiable credentials specification](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md). The identity provider API uses the [`window.postMessage()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) interface.
+
+Internet Identity is used as the identity provider in this tutorial.
+
+## Using the identity provider API
+
+To implement a relying party service, your canister must perform three steps:
+
+1. Load an identity provider such as Internet Identity: Open an identity provider that will act as the communication bridge between the relying party and the issuer API.
+
+2. Request credentials: Send a JSON RPC request to the issuer API for a user's credentials.
+
+3. Receive a response: Receive either a success or error message from the identity provider.
+
+### Step 1: Load the identity provider in a new window
```

Comment:
I would introduce the SDK and not describe the details.

```
### Use the Verifiable Credentials SDK

You can request credentials to issuers through identity providers with the JS client for relying parties NPM package [@dfinity/verifiable-credentials](https://www.npmjs.com/package/@dfinity/verifiable-credentials).

For more details on the steps that the library performs, check the [code](https://github.com/dfinity/verifiable-credentials-sdk/tree/main/js-library) of the [VC spec](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md).
```

Reply:
I think including the details and example code for the SDK is beneficial.

Reply:
I revised the first sentence but kept the SDK code example.

Reply:
The structure:

```
Step 1
Step 2
JS SDK
Step 3
Error message
```

Is confusing to me.

The SDK performs all steps: 1,2,3 and error management.

So, we can just:

```
JS SDK
```

Or if we want to explain what the SDK does:

```
JS SDK
Manual implementation
  Step 1
  Step 2
  Step 3
  Error management
```

But I believe that's unnecessary details that might lead to more confusing. If they want to manually implement it, they can look at the spec.